python/gLaSDI/examples/2DBurgerEqn/README.md

```diff
@@ -0,0 +1,20 @@
+The gLaSDI framework is applied to construct a non-intrusive reduced-order model of the **2D Burgers' Equation**.
+
+
+The data can be generated by `generate_data.ipynb`. Alternatively, some datasets are avaiable in `/usr/workspace/he10/data/2DBurgerEqn/`, which are used in the numerical examples in the paper. The datasets are generated from the parameter space constituted by the parameters of the initial condition, including the **width** ([0.9,1.1]) and the **amplitude** ([0.7,0.9]).
```

Comment:
This can be updated, since people will not be able to access the additional datasets.

Reply:
I will update it with more details for external usage.